### PR TITLE
Update auth0.md to include developer managed scopes UI steps

### PR DESCRIPTION
**Pull request by c3pko**
### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md [hunk 1]
```diff
@@ -88,9 +90,9 @@ Once you have Auth0 configured, you can set up the Dev Portal to use Auth0 for d
    {:.note}
    > **Note:** You can find the value for your `AUTH0_TENANT_SUBDOMAIN` by visiting **Settings** from the Auth0 sidebar and finding the **Tenant Name** in the **General** tab.
 
-6. Enter the **Client Token Audience** as the identifier value you set when configuring the API entity above
+6. Enter the **Client Token Audience** as the identifier value you set when configuring the API entity above. If you’re using Developer Managed Scopes, this vlaue should map to the audience field of your associated Auth0 API.
```
Comment (davidma415) [inline app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md:93]:
```suggestion
6. Enter the **Client Token Audience** as the identifier value you set when configuring the API entity above. If you’re using Developer Managed Scopes, this value should map to the audience field of your associated Auth0 API.
```

File: app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md [hunk 2]
```diff
@@ -88,9 +90,9 @@ Once you have Auth0 configured, you can set up the Dev Portal to use Auth0 for d
...
 
-7. Enter `openid` into the **Scopes** field
+7. Enter `openid` into the **Scopes** field. Note: If you’re using Developer Managed Scopes, check the “Developer Managed Scopes” box. Be sure to add the appropriate scopes (in addition to the mandatory openid scope) you wish your developers to be able to choose from, in string format like so: “openid”, “read:account_information”, “write:account_information”, etc. Note if you leave auth config scopes empty the developer will be able to choose from all valid scopes offered by that api.
```
Comment (davidma415) [inline app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md:95]: In Konnect, they can just enter it like `openid, read:account_information, write:account_information` - it just has to be comma delimited, but each scope doesn't need quotes around it

**Review by c3pko — COMMENTED** [review]
updating accuracy of information about adding scopes

File: app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md [hunk 3]
```diff
@@ -88,17 +91,19 @@ Once you have Auth0 configured, you can set up the Dev Portal to use Auth0 for d
    {:.note}
    > **Note:** You can find the value for your `AUTH0_TENANT_SUBDOMAIN` by visiting **Settings** from the Auth0 sidebar and finding the **Tenant Name** in the **General** tab.
 
-6. Enter the **Client Token Audience** as the identifier value you set when configuring the API entity above
+6. Enter the **Client Token Audience** as the identifier value you set when configuring the API entity above. If you’re using Developer Managed Scopes, this value should map to the audience field of your associated Auth0 API.
+
+7. Enter `openid` into the **Scopes** field.
 
-7. Enter `openid` into the **Scopes** field
+1. If you’re using developer-managed scopes, select the **Use Developer Managed Scopes** checkbox. Add the appropriate scopes (in addition to the mandatory `openid` scope) you want your developers to be able to choose from. For example: `openid`, `read:account_information`, and `write:account_information`.
```
Comment (cloudjumpercat) [inline app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md:98]:
```suggestion
1. If you’re using developer-managed scopes, select the **Use Developer Managed Scopes** checkbox. Add the appropriate scopes (in addition to the mandatory `openid` scope) you want your developers to be able to choose from. For example: `openid, read:account_information, write:account_information`.
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for updating these pages, @c3pko ! I'll approve this so it's ready for you to merge once the feature goes GA (it'll be published to the docs site as soon as you merge it).

**Pull request merged**